docker/k8s-config-with-om-monitors/agent.d/k8s_open_metrics_monitor.json

```diff
@@ -0,0 +1,18 @@
+{
+  "monitors":
+  [
+    {
+      module:  "scalyr_agent.builtin_monitors.kubernetes_openmetrics_monitor",
+      include_node_name: true,
+      include_cluster_name: true,
+      logger_include_node_name: false,
+      verify_https: false,
```

Comment:
Since the default value for that option is True, just want to make sure that this is expected configuration.

Reply:
Will add a comment to the config to clarify that.